### PR TITLE
Refine the code

### PR DESCRIPTION
**Pull request by ShengchenJ**

### CODE REVIEW

**Review by zhiweij1 — APPROVED** [review]

File: clang/runtime/dpct-rt/include/dpct/codepin/detail/json.hpp [hunk 1]
```diff
@@ -215,17 +221,15 @@ inline value &object::operator[](std::string &&key) {
 }
 
 inline const value &object::get(const std::string &key) {
-  if (!contains(key)) {
-    error_exit("The " + key + " needs to define in the schema.");
-  }
-  return obj_map[key];
+  if (contains(key))
+    return obj_map[key];
+  error_exit("[JSON Parser]: The key " + key + " is not in the JSON.\n");
```
Comment (AndyCHHuang) [inline clang/runtime/dpct-rt/include/dpct/codepin/detail/json.hpp]: Parsing json to cpp failed. Missing key "key" in the json file.

File: clang/runtime/dpct-rt/include/dpct/codepin/detail/json.hpp [hunk 2]
```diff
@@ -237,7 +241,8 @@ inline bool object::contains(const std::string &str) {
 inline bool json_parser::parse_string(std::string &ret_str) {
   while (peek() != '"') {
     if (cur_p == end) {
-      error_exit("The string needs \" to wrap it. Please check the json "
+      error_exit("[JSON Parser]: Parse string needs \" to wrap it. Please "
```
Comment (AndyCHHuang) [inline clang/runtime/dpct-rt/include/dpct/codepin/detail/json.hpp]: key value of a json need to be wrapped in \". Please check the json file format.

File: clang/runtime/dpct-rt/include/dpct/codepin/detail/json.hpp [hunk 3]
```diff
@@ -258,13 +263,12 @@ inline bool json_parser::parse_num(char first, int64_t &out) {
...
     val += next();
   }
-  // Check the value;
   char *end;
   out = strtoimax(val.c_str(), &end, 10);
   if (end == (val.c_str() + val.size())) {
     return true;
   }
-  error_exit("The number format is not correct; please check; \n");
+  error_exit("[JSON Parser]: Parse number failed, please check the JSON.\n");
```
Comment (AndyCHHuang) [inline clang/runtime/dpct-rt/include/dpct/codepin/detail/json.hpp]: key

File: clang/runtime/dpct-rt/include/dpct/codepin/detail/json.hpp [hunk 4]
```diff
@@ -132,7 +139,7 @@ class value {
       // To do
       break;
     default:
-      error_exit("The type is not supported. Please double check");
+      error_exit("[JSON Parser]: Parsed unkown value type.\n");
```
Comment (AndyCHHuang) [inline clang/runtime/dpct-rt/include/dpct/codepin/detail/json.hpp]: key

**Review by AndyCHHuang — APPROVED** [review]

**Pull request merged**